src/components/dashboard/users/SelectElement.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from 'react';
+import globalize from '../../../scripts/globalize';
+
+const createSelectElement = ({ className, label, option }) => ({
+    __html: `<select
+        class="${className}"
+        is="emby-select"
+        label="${label}"
+        >
+        ${option}
+    </select>`
+});
+
+type IProps = {
+    className?: string;
+    label?: string;
+    currentProviderId: string;
+    providers: any
```

Comment:
The type should probably be `any[]`

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3070/commits/1a66afa6104ce9c68bac300626a35f5d1426a15f